src/__tests__/actions/rating/createRate.test.js

```diff
@@ -1,31 +1,18 @@
 import React from 'react';
-import configureMockStore from 'redux-mock-store';
```

Comment:
why you changed this file while you didn't work on it

Reply:
As I introduced a file for mocking the store, I found that it would good to use the same mocked store

Reply:
Good